src/test/java/ca/yorku/cse2311/tab2pdf/parser/DoubleBarParserTest.java

```diff
@@ -0,0 +1,110 @@
+package ca.yorku.cse2311.tab2pdf.parser;
```

Comment:
Remove `println()`s here, Test case output should be clean!

Additionally, since two possible exceptions are thrown by `parse()` (`ParseException` and `NumberFormatException`), this test should be split up into two. One test should expect a `ParseException`, and one should expect a `NumberFormatException`. These tests should fail if the wrong exception is thrown